### ISSUE TITLE
Plugin sets "useFrameworks" to "static"

### ISSUE THREAD

**Issue by Noitham** [braze-inc/braze-expo-plugin#13]
### Which Platforms?

iOS

### Which React Native Version?

0.70.5

### Which @braze/expo-plugin version?

1.0.0

### Which @braze/react-native-sdk SDK version?

1.41.0

### Repro Rate

100

### Steps To Reproduce

When running the `expo prebuild` command
```
"ios.useFrameworks": "static"
```
is added on the `Podfile.properties.json` file.

This, will break builds that are using Flipper, for example.

I think this should be listed somewhere in the docs.

### Expected Behavior

Docs should warn of plugin setting `useFramework` to `static` 

### Actual Incorrect Behavior

```
"ios.useFrameworks": "static"
```

**Comment by wesleyorbin** [comment]
Hi @Noitham. It looks like we need a small change on the React Native SDK's podspec to be able to remove this setting. Once that's released, we'll push an update to the Expo plugin that removes it.

**Comment by wesleyorbin** [comment]
Hi @Noitham. We just released v1.1.0 of the plugin which removes the usage of static frameworks. Note that you'll need to update the React Native SDK to [2.1.0](https://github.com/braze-inc/braze-react-native-sdk/blob/master/CHANGELOG.md#210) in order to take advantage of this update.

**Comment by Noitham** [comment]
Glad to hear, thanks for the quick response & solution 🙌
